pydrive2/fs/spec.py

```diff
@@ -542,6 +542,43 @@ def cp_file(self, lpath, rpath, **kwargs):
             buffer = io.BytesIO(stream.read())
             self.upload_fobj(buffer, rpath)
 
+    @_gdrive_retry
+    def mv(self, path1, path2, recursive=False, maxdepth=None, **kwargs):
+
+        if maxdepth:
+            raise NotImplementedError("Max depth move is not supported")
+
+        if self.isdir(path1) and not recursive:
+            raise AttributeError(
+                "Cannot move a directory without recursive flag"
+            )
+
+        file1_name = posixpath.basename(path1)
```

Comment:
Let's name things `src` / `dst`. Since they are not files in all cases and it's more descriptive.